index.js

```diff
@@ -49,11 +56,71 @@ class Books {
     this.saveToLocalStorage();
     this.displayBooks();
   };
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227219091-aa2f6f04-c8c7-45ea-87fa-9e08f7d080e6.png)

**[OPTIONAL]**

- [ ] It would be nice if you make all your commits meaningful and capitalize to make your pull request professional 😊

Reply:
I shall implement the changes asked.